admin/angel/Core.c

```diff
@@ -198,6 +227,11 @@ void Core_init(struct Allocator* alloc,
     Iface_plumb(&nc->tunAdapt->ipTunnelIf, &ipTunnel->tunInterface);
     Iface_plumb(&nc->upper->ipTunnelIf, &ipTunnel->nodeInterface);
 
+#ifndef win32
+    struct FileNo_admin* fileno = FileNo_admin_new(admin, alloc, eventBase,
+                                                 logger, eh, onFileNoReceived);
+#endif
```

Comment:
Should be called FileNo_admin_register() and should return `void` like the others.